### PR TITLE
EPS-9: Configure and initialize database

### PR DESCRIPTION
**Pull request by vladqf**
Link to jira ticket: https://ifortex-team-java1.atlassian.net/browse/EPS-9?atlOrigin=eyJpIjoiMzY1NWJiMTQ1MDBlNDM3NGE5Y2UyYzc5NWIxYmI3MzEiLCJwIjoiaiJ9

Set up a database using docker-compose, made a script to create a database schema and fill it with data.

### CODE REVIEW

File: docker-compose/db-init/V1__initial.sql [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                                    WHERE s.id IN (SELECT s2.id
+                                                   FROM students s2
+                                                   ORDER BY random()
+                                                   LIMIT sc.student_count))
+INSERT
+INTO m2m_student_course (course_id, student_id)
+SELECT course_id,
+       student_id
+FROM course_student_enrollments
+ORDER BY course_id, student_id;
```
Comment (PavelShynkarou) [inline docker-compose/db-init/V1__initial.sql:69]: Did you test it? Does it work properly?

**Review by PavelShynkarou — CHANGES_REQUESTED** [review]
Looks like your Jira ticket in the wrong state. Please fix it.

**Review by AndreySemeynikovIF — APPROVED** [review]

**Review by AndreySemeynikovIF — APPROVED** [review]

File: docker-compose/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    environment:
+      POSTGRES_USER: ifortex
+      POSTGRES_PASSWORD: ifortex
+      POSTGRES_DB: education_db
+      POSTGRES_DBINIT_ARGS: "--encoding=UTF8 --locale=en_US.UTF8"
+    volumes:
+      - ./db-init:/docker-entrypoint-initdb.d
+
+    
+
```
[the remark this answers is not in the document]
Comment (vladqf) [reply docker-compose/docker-compose.yml]: Removed all empty lines except one.

File: docker-compose/db-init/V1__initial.sql (hunk 1 above)
Comment (vladqf) [reply docker-compose/db-init/V1__initial.sql:69]: I tested it and it works properly

**Comment by vladqf** [comment]
> Looks like your Jira ticket in the wrong state. Please fix it.

Fixed

**Review by PavelShynkarou — APPROVED** [review]

**Review by AndreySemeynikovIF — APPROVED** [review]

**Review by AndreySemeynikovIF — APPROVED** [review]

**Review by AndreySemeynikovIF — APPROVED** [review]

**Review by PavelShynkarou — APPROVED** [review]

**Pull request merged**